ui/src/app/admin/_components/personTable.tsx

```diff
@@ -0,0 +1,284 @@
+            </div>
+            <div className="flex flex-col md:flex-row md:justify-between items-center mb-4">
+                <SearchInput />
+                <label>
+                    <div>
+                        Check All
+                        <input
+                            className="mx-2"
+                            type="checkbox"
+                            name="checkAll"
+                            checked={userInfo === undefined ? dummyBool : table.getIsAllPageRowsSelected()}
+                            onChange={
+                                userInfo === undefined
+                                    ? () => setDummyBool(!dummyBool)
+                                    : table.getToggleAllPageRowsSelectedHandler()
+                            }
+                        />
+                        <Button
+                            className="rounded-[-0.25rem] rounded-r-[0.25rem]"
+                            variant="destructive"
+                            onClick={userInfo === undefined ? () => void 0 : handleRemove}
+                        >
+                            Remove
+                        </Button>
+                    </div>
+                </label>
+            </div>
+            {validUid === 'FAILURE' && searchUid !== undefined ? (
+                <Alert variant="destructive" className="w-fit mb-7">
+                    <AlertDescription>{searchUid} is not in any grouping.</AlertDescription>
+                </Alert>
+            ) : (
+                ''
+            )}
+            {validUid === undefined && searchUid !== '' ? (
+                <Alert variant="destructive" className="w-fit mb-7">
+                    <AlertDescription>
+                        There was an error searching for {searchUid}. <br />
+                        Please ensure you have entered a valid UH member and try again.
+                    </AlertDescription>
+                </Alert>
+            ) : (
+                ''
+            )}
+            {searchUid === '' ? (
+                <Alert variant="destructive" className="w-fit mb-7">
+                    <AlertDescription>You must enter a UH member to search.</AlertDescription>
+                </Alert>
+            ) : (
+                ''
+            )}
+            <Table className="relative overflow-x-auto">
+                <TableHeader>
+                    {table.getHeaderGroups().map((headerGroup) => (
+                        <TableRow key={headerGroup.id}>
+                            {headerGroup.headers.map((header) => (
+                                <TableHead
+                                    key={header.id}
+                                    onClick={
+                                        header.column.id === 'name'
+                                            ? header.column.getToggleSortingHandler()
+                                            : () => void 0
+                                    }
```

Comment:
You should be able to disable sorting for a certain column in your person-table-columns.tsx file. Then you can just use the getToggleSortingHandler() without this ternary,